modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -17,6 +17,11 @@ endif::[]
 [id="nw-egressnetworkpolicy-about_{context}"]
 = How an egress firewall works in a project
 
+[NOTE]
```

Comment:
My only suggestion here is to move your note below the bulleted list. Starting the topic with a note, instead of the abstract might affect how search engines crawl this content. I suspect a search hit would likely display the content of the note ahead of your abstract.

Reply:
Addressed